src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.scss

```diff
@@ -0,0 +1,31 @@
+@import "scss/constants";
+
+$gridTotalRows: var(--map-partygoer-overlay-total-rows, 1);
+$gridTotalColumns: var(--map-partygoer-overlay-total-columns, 1);
+$row: var(--map-partygoer-overlay-row, 0);
+$column: var(--map-partygoer-overlay-column, 0);
+
+$avatarWidth: calc(100% / #{$gridTotalRows});
+$avatarHeight: calc(100% / #{$gridTotalColumns});
+$avatarTop: calc((#{$row} - 1) * #{$avatarWidth});
+$avatarLeft: calc((#{$column} - 1) * #{$avatarHeight});
```

Comment:
[FYI] Calling out this pattern of using CSS `--variables` to pass dynamic content from the JS into SCSS, so that we can keep our styles in SCSS rather than inline styles.

(cc // @sparkletown/engineering FYI: you'll likely see more of this pattern moving forward)